### PR TITLE
Deactivate

### PR DESCRIPTION
**Pull request by rbillings**

### CODE REVIEW

**Comment by rbillings** [comment]
Please review @karlht , @stephendonner - checks have failed because it tried to run tests on deleted files.

File: README.md [hunk 1]
```diff
@@ -1,328 +1,16 @@
-oneanddone
-==========
+# One and Done
 
-One and Done is written with [Django][django].
+This repository holds code and automation that were used for Mozilla's [One and Done](https://oneanddone.mozilla.org/) website.
```
Comment (stephendonner) [inline README.md]: Should this be "This repository held code"?

**Comment by stephendonner** [comment]
While I've removed the Travis-CI integration/webhook, we'll still need to 1) push another change to the repo (assuming that'll obviate the former "required-status check"? and/or 2) change the repo in GitHub to not require that former status check.

Looking into it still.

File: README.md (hunk 1 above)
Comment (rbillings) [inline README.md]: I don't think so as the code is still there - you just have to look for it.

Comment (rbillings) [inline README.md]: Also, it was written to match https://github.com/mozilla/moz-grid-config/pull/161/files

**Comment by rbillings** [comment]
@stephendonner - do we really need to make those changes for Travis as no more code will ever be pushed?

**Comment by stephendonner** [comment]
@rbillings with the way it was set up, I believe so.  I was thinking we should remove the https://github.com/rbillings/oneanddone/blob/ff64f1886e648839b70c6af43fb25b14eda1759c/.travis.yml file.  Might be good for @karlht @bobsilverberg @davehunt to weigh in.  My past decommissions of projects have removed every file from the repo, leaving only the README as a pointer to why, so not sure if we should be doing the same here?

**Comment by stephendonner** [comment]
e.g. https://github.com/mozilla/marketplace-tests/pull/761 and from @justinpotts ' https://github.com/mozilla/Affiliates-Tests/pull/139

**Comment by rbillings** [comment]
I removed all of the files beginning with "." that I missed on the first round.

**Comment by stephendonner** [comment]
For posterity, I also disabled the requirement that the master branch have a passing Travis CI build before merging pulls, which is no longer possible, given the nature of the changes (code removal).

screen shot 2016-10-31 at 12 08 03 pm

**Review by stephendonner — CHANGES_REQUESTED** [review]
Thanks, @rbillings - these changes look good to me, but I think you can probably also remove .gitignore.

**Comment by stephendonner** [comment]
(And let me know if you need help with any of the steps in https://wiki.mozilla.org/QA/Execution/Web_Testing/Repositories#Deactivating_a_repository.)

**Comment by rbillings** [comment]
Removed the last sneaky .gitignore file.

**Review by stephendonner — APPROVED** [review]

**Comment by stephendonner** [comment]
Thanks; r+ - leaving open to give @karlht an opportunity to r+/merge, etc.

**Comment by karlht** [comment]
@stephendonner This looks fine to me -- you may have the honours.  Merge at will.

**Pull request merged**